README.md

```diff
@@ -1,2 +1,3 @@
 # team-one-project-one
 Team #1's first project.
+Anthony was here first 👍
```

Comment:
🤔 hummm looks interesting.

Reply:
looks to me like it's ready to be turned in